app/logo.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+from cairosvg import svg2png
+from flask import abort, Blueprint, current_app, send_file
+from io import BytesIO
+from notifications_utils.statsd_decorators import statsd
+
+from app.preview import get_logo_from_filename
+
+logo_blueprint = Blueprint('logo', __name__)
+
+
+@logo_blueprint.route("/<logo>.svg.png", methods=['GET'])
+@statsd(namespace="template_preview")
```

Comment:
Should this endpoint require authentication (so only the admin app can talk to it)? I only removed it to make it easier to test locally.

```suggestion
@statsd(namespace="template_preview")
@auth.login_required
```

Reply:
sure